frontend/src/Components/NotificationsPopup.tsx

```diff
@@ -152,7 +169,7 @@ export default function NotificationsPopup(props: NotificationsPopupProps) {
                     })}
                 </div>
                 <div className={styles.buttons}>
-                    <button className={styles.buttonClear} onClick={handleClearAll}>Очистить</button>
+                    <button className={styles.buttonClear} onClick={handleClearAll}>Очистить прочтенные</button>
```

Comment:
и ведь попробуй объяснить это кому-то, кому русский не родной язык.
```suggestion
                    <button className={styles.buttonClear} onClick={handleClearAll}>Очистить прочитанные</button>
```